packages/grafana-ui/src/components/SimpleSelect/SimpleSelect.tsx

```diff
@@ -0,0 +1,59 @@
+import { useCombobox } from 'downshift';
+import React, { useState } from 'react';
+
+import { SelectableValue } from '@grafana/data';
+
+import { Icon } from '../Icon/Icon';
+import { Input, Props as InputProps } from '../Input/Input';
+
+interface SimpleSelectProps
+  extends Omit<InputProps, 'width' | 'prefix' | 'suffix' | 'value' | 'addonBefore' | 'addonAfter'> {
+  onChange: (val: SelectableValue) => void;
+  value: SelectableValue;
```

Comment:
My gut feeling is that `value` should be a primative (`string` or `string | number`), to make it easier to pick the correct value. I think making it be an object begins an awkward slipery slope of

```ts
<SimpleSelect
  value={{ label: "Hello!", value: "hello" }}
  options={[
    { label: "Hello!", value: "hello" },
    { label: "World!", value: "world" },
  ]}
/>
```

Which value, if any, is currently selected? 🤔 

```ts
<SimpleSelect
  value={{ label: "foo!", value: "hello" }}
  options={[
    { label: "Hello!", value: "hello" },
    { label: "World!", value: "world" },
  ]}
/>
```

What about now?


I think restricting `value` to just `string`, we remove a bunch of ambiguity.

Reply:
I agree it makes sense. I think with a good example it will be hard to miss too.